test/test.js

```diff
@@ -273,6 +273,18 @@ describe('babel-plugin-transform-vue-jsx', () => {
     expect(vnode.tag).to.equal('div')
     expect(vnode.children[0].text).to.equal('test')
   })
+
+  it('should handle special properties', () => {
+    const noop = _ => _
+    const vnode = render(h => (
+      <input value="value" />
+    ))
+    expect(vnode.data.domProps.value).to.equal('value')
+    const vnode2 = render(h => (
+      <input value="value" type="button" />
```

Comment:
Just occurs to me that a static `value` attribute should probably be treated as the initial value only and thus be transpiled into `attrs`. Is there a way to differentiate this with `value={...}`? Probably just checking if the node's bound value is a literal string?